### DISCUSSION TITLE
IP webcam no camera message

### DISCUSSION THREAD

**Discussion by EyeVisionsNL** [dinki/View-Assist#97] [General]
When using the IP webcam program, I get a message at the top of the screen from the program that it cannot use the camera. Even if I set audio only in settings and, for example, let him use my front cameras. Also I can't uncheck the overlay like you do in video. Not even as a pro user. Am I the only one with this problem?

**Comment by mngarchow** [comment]
Which OS and/or tablet model are you using? Perhaps it is something related to your hardware and not the app itself?

**Comment by dinki** [comment]
Agreed.  I have seen others mention this as well.  

The best place for assistance is definitely Discord if you can join.

Some good news is that one of our members has created his own app and integration that is FAR superior to IPWebCam and RTPMic.  It is called HassMic.  He is still working on extending it to also include media player capabilities but I have been using it in production for more than a month and it is straight up awesome.   It will become the recommended application after the mediaplayer work is finished (hopefully soon).  I would suggest you install it and give it a try instead of using Stream Assist and IPWebcam.

https://github.com/jeffc/hassmic

**Comment by dinki** [comment]
Problem solved.  Yeah!
